ccbc-domain/src/main/java/io/coin/ccbc/domain/CoinHistory.java

```diff
@@ -35,7 +35,19 @@
   @Column(name = "balance", nullable = false, precision = 78)
   private BigInteger balance;
 
-  public static CoinHistory collected(Address wallet, Coin coin, BigInteger balance) {
-    return new CoinHistory(wallet, coin, balance);
+  private CoinHistory(Address wallet, Coin coin, BigInteger balance, LocalDateTime time) {
+    super(time);
```

Comment:
어떤 필드가 들어가는지 명시적으로 표현하기 위해
```suggestion
    super(collectedAt);
```
이렇게는 어떨까용??